static/sass/custom/_search-results.scss

```diff
@@ -89,6 +89,33 @@
       margin-top: -0.5rem;
     }
   }
+
+  .p-contextual-menu {
+    &__toggle {
+      text-transform: capitalize;
+      
+      &--active {
+        border-color: $color-focus;
+      }
+    }
+
+    &__dropdown {
+      min-width: 19rem;
+      padding: 0.5rem 0;
+
+      .p-list__item {
+        padding: 0.5rem 1rem;
+      }
+
+      .series-tag:last-child {
```

Comment:
This styling should live with in the series-tag component

Reply:
Thanks, moved.